docs/README.md

```diff
@@ -1,3 +1,4 @@
 # Documentation
 
+* [Getting Started](./getting-started.md)
```

Comment:
Nit-pick: Since the other Markdown file uses "regular" capitalization, let's align this one and name it `Getting-started.md`?